card/lib/card/loader.rb

```diff
@@ -121,25 +131,26 @@ def load_implicit_sets_from_source set_pattern
 
       def prepare_tmp_dir path
         if rewrite_tmp_files?
-          p = Cardio.paths[ path ]
+          p = Card.paths[ path ]
           if p.existent.first
             FileUtils.rm_rf p.first, :secure=>true
           end
           Dir.mkdir p.first
         end
       end
-      
+
       def rewrite_tmp_files?
         if defined?( @@rewrite )
           @@rewrite
         else
-          @@rewrite = !( Rails.env.production? and Cardio.paths['tmp/set'].existent.first )
+          @@rewrite = !( Rails.env.production? and Card.paths['tmp/set'].existent.first )
         end
       end
 
       def load_dir dir
         Dir[dir].sort.each do |file|
 #          puts Benchmark.measure("from #load_dir: rd: #{file}") {
+warn "load file #{file}"
```

Comment:
Will remove, debug code.  I think I was debugging the build and it was another case of stuff not loading because seeds didn't load, so modules don't load.